etc/env/file_based/overrides/legacy-new.yaml

```diff
@@ -0,0 +1,7 @@
+db:
+  merkle_tree:
+    mode: LIGHTWEIGHT
+experimental_vm:
+  state_keeper_fast_vm_mode: NEW
+mempool:
+  delay_interval: 50
```

Comment:
nit: config itself has nothing to do with legacy, I would change name to loadtest-new.yaml and loadtest-old.yaml